assets/vizList.js

```diff
@@ -1,83 +1,202 @@
-      "link": "https://public.tableau.com/profile/sarah.rodriguez#!/vizhome/ReturnsDashboard_15958669451740/Shelter?publish=yes",
-      "url": "https://public.tableau.com/views/ReturnsDashboard_15958669451740/Shelter?:language=en&:display_count=y&publish=yes&:origin=viz_share_link"
-    },
-    {
-      "section": 3,
-      "section_title": "Housing",
-      "layer": 1,
-      "visualization_description": "Change in the total number of beds for housing",
-      "link": "https://public.tableau.com/profile/sarah.rodriguez#!/vizhome/Changeinthetotalnumberofbedsusedtohousepeople/HousingandShelter3?publish=yes",
-      "url": "https://public.tableau.com/views/Changeinthetotalnumberofbedsusedtohousepeople/HousingandShelter3?:language=en&:display_count=y&publish=yes&:origin=viz_share_link"
-    },
-    {
-      "section": 3,
-      "section_title": "Housing",
-      "layer": 2,
-      "visualization_description": "Change in the total number of people who exit to homelessness to housing",
-      "link": "https://public.tableau.com/profile/sarah.rodriguez#!/vizhome/Exits_15930134501040/Exits?publish=yes",
-      "url": "https://public.tableau.com/views/Exits_15930134501040/Exits?:language=en&:display_count=y&publish=yes&:origin=viz_share_link"
-    },
-    {
-      "section": 3,
-      "section_title": "Housing",
-      "layer": 3,
-      "visualization_description": "Comparison of the demographics of people who are housed and sheltered",
-      "link": "https://public.tableau.com/profile/sarah.rodriguez#!/vizhome/HousingDashboard_15958669097400/Housing?publish=yes",
-      "url": "https://public.tableau.com/views/HousingDashboard_15958669097400/Housing?:language=en&:display_count=y&publish=yes&:origin=viz_share_link"
-    },
-    {
-      "section": 4,
-      "section_title": "Returns to Homelessness",
-      "layer": 1,
-      "visualization_description": "Change in the total number of people who return to homelessness",
-      "link": "https://public.tableau.com/profile/sarah.rodriguez#!/vizhome/ReturnstoHomelessness_15919382098890/Returns-total?publish=yes",
-      "url": "https://public.tableau.com/views/ReturnstoHomelessness_15919382098890/Returns-total?:language=en&:display_count=y&publish=yes&:origin=viz_share_link"
-    },
-    {
-      "section": 4,
-      "section_title": "Returns to Homelessness",
-      "layer": 2,
-      "visualization_description": "Comparison of the demographics of people who return to homelessness and people who are housed",
-      "link": "https://public.tableau.com/profile/sarah.rodriguez#!/vizhome/ReturnsDashboard_15958669451740/Return?publish=yes",
-      "url": "https://public.tableau.com/views/ReturnsDashboard_15958669451740/Return?:language=en&:display_count=y&publish=yes&:origin=viz_share_link"
-    }
-   ];
-   
+  {
+    section: 1,
+    section_title: "Newly Homeless",
+    layer: 1,
+
+    visualization_description:
+      "Change in the number of people who are newly homeless",
+    info: `
```

Comment:
like i mentioned in the comment above, adding the tooltip content info is def a tradeoff. the pro is that we dont have to worry about importing more content objects, the con is that putting the html here takes up significant space and might make the object a bit harder to skim through. i think its ok for now tho?